### DISCUSSION TITLE
How to use conditional styles?

### DISCUSSION THREAD

**Discussion by 60pfennig** [Mokshit06/typewind#54] [Q&A] [answered]
Hi,
How can I create coditional styles? In tailwind I could use template literals to render the classNames I want to use. But how can I do this in Typewind?
Thanks :)

**Comment by kavinvalli** [comment, marked as answer]
It just works like any other variable inside a template literal. 
```js
const className = `${tw.some_common_class} ${active ? tw.class_you_want : tw.some_other_class}`
```
That being said, I recommend using the clsx and tailwind-merge packages to work with conditional classes

ps: I wrote this answer on my phone so please ignore any typos/syntax errors

**Reply by 60pfennig** [reply]
thank you for the fast and good reply!
